kong/plugins/ip-restriction/handler.lua

```diff
@@ -21,18 +59,18 @@ end
 function IpRestrictionHandler:access(conf)
   IpRestrictionHandler.super.access(self)
   local block = false
-  local remote_addr = ngx.var.remote_addr
+  local remote_addr = ngx.var.binary_remote_addr
```

Comment:
It would make sense to rename this variable to `bin_remote_addr` or `binary_remote_addr` as well, eventually.